__test__/server/texter.test/texter.test.js

```diff
@@ -156,11 +156,12 @@ it("should be able to receive a response and reply (using fakeService)", async (
       expect.objectContaining({
         send_status: "DELIVERED",
         text: `responding to ${message.text}`,
-        user_id: testTexterUser.id,
+        user_id: null,
```

Comment:
while adding tests, this bug showed up. This is_from_contact=true shouldn't have a user_id since it's allegedly sent from the contact.